README.md

```diff
@@ -37,3 +37,10 @@ _Past Maintainers_
 
 - Jorge Vicente Cantero - [@jvican](https://github.com/jvican)
 - Ólafur Páll Geirsson - [@olafurpg](https://github.com/olafurpg)
+
+## Protocol changelog
+
+### 2.1.1
```

Comment:
2.1.1 does not yet exist so I find it weird to mention it here.

Also if we add something to the protocol, should it be a minor version bump? => 2.2.0

```suggestion
### 2.1.1 (future version)
```

Or

```suggestion
### Unreleased
```

Reply:
I agree, that's why I opened https://github.com/build-server-protocol/build-server-protocol/pull/606